content/en/platform/corda/5.0-dev-preview-1/tutorials/building-cordapp/c5-basic-cordapp-flows.md

```diff
@@ -449,6 +449,179 @@ data class CreateBoardingTicketInitiator @JsonConstructor constructor(private va
 }
 ```
 
+## Write the `GiftVoucherToFriend` flow
+
+In Corda, you can transfer a state to someone else. The `GiftVoucherToFriend` flow lets you transfer the ownership of the issued voucher to another person.
+
+Now that you've written the `CreateAndIssueMarsVoucher` and `CreateBoardingTicket` flows, try writing the `GiftVoucherToFriend` flow.
+
+You will need these variables:
+
+* `voucherID`
```

Comment:
I'd add a description of the variables to explain why they are needed/what they will do in the flow.

Reply:
@peterli-r3 could you provide that?